src/index.js

```diff
@@ -39,6 +40,8 @@ const optionsToConfig = (
         fragments,
         schema,
         errors,
+        allObjectTypes: null,
+        path: [],
```

Comment:
The addition of these items makes me think that we may want to rename `config` to `context` in the future.  No action req'd.